surfactant/relationships/dotnet_relationship.py

```diff
@@ -75,8 +75,16 @@ def establish_relationships(
     #    - application base + culture + assembly name directories
     #    - privatePath directories from a probing element, combined with culture/appbase/assemblyname (done before the standard probing directories)
     #    - the location of the calling assembly may be used as a hint for where to find the referenced assembly
+    def chained_lists(*it):
+        for iterab in it:
+            yield from iterab
+
     if "dotnetAssemblyRef" in metadata:
-        for asmRef in metadata["dotnetAssemblyRef"]:
+        combinedRefs = [metadata["dotnetAssemblyRef"]]
+        if "dotnetImplMap" in metadata:
+            combinedRefs.append(metadata["dotnetImplMap"])
```

Comment:
What were the URLs/doc links that you referenced for info on how .NET determines where to look for the DLLs listed in ImplMap?

Reply:
I was mistaken about unmanaged code loading being the same as loading assemblies. I got my sources mixed up, but I found the one that relates to ImplMap (PInvoked) libraries and included it. From what it looks like, if it's not the assembly/application directory, it's the system32 folder (default unless someone overrides the loading function).